packages/spaces/src/SpacesLink.js

```diff
@@ -259,7 +261,13 @@ const Link = ({
       >
         {!stacked && favoriteIcon}
         {appIcon}
-        {icon && <Icon name={icons.navigation} />}
+        {icon && type.toUpperCase() === 'FILE' ? (
+          <AvLink target="_blank" href={restLink.url}>
```

Comment:
I don't think the Codecov diff is preventing a merge, but if you have time can you add a test for this new logic?